wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -95,7 +95,20 @@ public static Collection<AwsWrapperProperty> allProperties() {
     return PROPS_BY_NAME.values();
   }
 
-  public static void removeAll(Properties props) {
-    PROPS_BY_NAME.keySet().forEach(props::remove);
+  public static void removeAllExceptCredentials(Properties props) {
+    String user = props.containsKey(PropertyDefinition.USER.name)
```

Comment:
You can simplify the logic:
props.getOrDefault("key", (String)null);

Reply:
Thanks for the tip!